tools/plot.py

```diff
@@ -15,14 +15,16 @@ def boxes(time, vars, *boxes, axs=None, label=None, **kwargs):
         A list of axes to plot the variables in. If not given, a new figure and axes are created.
     label : str, optional
         Some custom text to add to the legend.
+    height : float, optional
+        Height modifier to edit the height of the figure
     **kwargs
         Keyword arguments to pass to the pyplot.plot function.
     """
     if isinstance(vars, str):
         vars = [vars]
         
     if axs is None:
-        fig, axs = plt.subplots(len(vars), 1, figsize=(8, 1.7 * len(vars)), sharex=True, constrained_layout=True)
+        fig, axs = plt.subplots(len(vars), 1, figsize=(8, (1.7 * len(vars)) + height), sharex=True, constrained_layout=True)
```

Comment:
This works, but might be more elegant as `figsize=(8, (panel_height * len(vars))`, then you could specify the default value of `panel_height` to be 1.7 in the function definition?

You could also add a `width` argument.

Reply:
Done both of these now